### ISSUE TITLE
The v of the version should be included in the link in search result listings.

### ISSUE THREAD

**Issue by sigurdm** [dart-lang/pub-dev#7244]
The v of the version should be included in the link in search result listings.

![image](https://github.com/dart-lang/pub-dev/assets/8613953/01375f3e-e48e-4e0a-9753-209f12ecdac8)
